python/hail/representation/genomeref.py

```diff
@@ -0,0 +1,199 @@
+from hail.java import handle_py4j, jiterable_to_list
+from hail.typecheck import *
+from hail.representation.interval import Interval
+from hail.utils import wrap_to_list
+from hail.history import *
+
+
+class GenomeReference(HistoryMixin):
+    """An object that represents a genome reference.
```

Comment:
`reference genome.`

You could hyperlink to: https://en.wikipedia.org/wiki/Reference_genome